app/assets/javascripts/redux/modules/models/promotionalTweets.js

```diff
@@ -25,25 +25,6 @@ export const initialState = {
   loading: false,
 };
 
-function handlePromoTweetSuccessResponse(state, action) {
-  const normalizedPromoTweet = normalizeListenSignalsResponse(action.payload);
-  const promotionalTweetResponse = _.get(normalizedPromoTweet, 'entities.promotionalTweet', {});
-  const id = _.findLastKey(promotionalTweetResponse);
-  const promotionalTweet = promotionalTweetResponse[id];
-
-  return {
-    ...state,
-    data: {
-      ..._.get(state, 'data', {}),
-      [id]: {
-        ..._.get(state, `data.${id}`, {}),
-        ...promotionalTweet,
-        loaded: true,
```

Comment:
This was mostly right. We just don't need to state loaded true or get the previous state of the promotional tweet.